pkgs/development/cuda-modules/cuda/overrides.nix

```diff
@@ -328,8 +331,11 @@ filterAndCreateOverrides {
         prevAttrs.postPatch or ""
         + ''
           for path in $rmPatterns; do
-            rm -r "$path"
+            rm -r $path
           done
+        ''
+        + ''
+          patchShebangs nsight-systems
```

Comment:
Am I missing something, or could this just be combined into one multi-line string instead of concatenating two multi-line strings?